charts/cryostat/templates/deployment.yaml

```diff
@@ -208,10 +208,12 @@ spec:
               value: {{ include "cryostat.cookieSecret" . }}
             - name: OAUTH2_PROXY_EMAIL_DOMAINS
               value: "*"
-            - name: OAUTH2_PROXY_HTPASSWD_FILE
-              value: /etc/oauth2_proxy/auth_proxy_htpasswd
```

Comment:
In short from above, don't remove this piece. Wrap it in `{{ if .Values.authentication.basicAuth.enabled }}`, so that we set this environment variable telling the proxy to load the htpasswd file only when the user has enabled basic auth.